### DISCUSSION TITLE
Loop buttons

### DISCUSSION THREAD

**Discussion by 0xHouss** [TomSchimansky/CustomTkinter#1025] [General]
```
from customtkinter import *

app = CTk()

app.geometry("500x500")

for i in range(3):
    btn = CTkButton(app, text=i, hover_color="#1F6AA5")
    btn.pack()
    btn.bind("<Enter>", lambda _: [btn.configure(fg_color="yellow", text_color="red")])
    btn.bind("<Leave>", lambda _: [btn.configure(fg_color="#1F6AA5", text_color="white")])

app.mainloop()
```

this is my example code where i packed 3 buttons using a for loop and i want to make a custom hover with an <Enter> and <Leave> bind but the problem is that when i hover on any of the buttons it's always the last one who reacts: 
![image](https://user-images.githubusercontent.com/67522466/211371448-01c86a0a-f884-4110-ab39-21d6246aaae2.png)
![image](https://user-images.githubusercontent.com/67522466/211371497-9bc1c654-cbef-496c-98d9-3b9adc27639b.png)

how can i make it work?

**Comment by Akascape** [comment]
@jl-houss Refer these examples:
https://github.com/TomSchimansky/CustomTkinter/discussions/909#discussioncomment-4464367
